nio_cli/nio_add_blocks/main.py

```diff
@@ -118,7 +118,30 @@ def __init__(self, args):
         self.args = args
 
     def perform(self):
+        ''' Clone down the project_template repository
+
+        If the installed version of nio matches a release in the
+        project_template repo, then use that tag.
+        '''
         BlockAdder.add(self.args.project_name, True, self.args.https)
+        version = self._nio_version()
+        print('n.io version: {}'.format(version))
+        if version:
+            try:
+                subprocess.call('cd ./{} && git checkout tags/{}'.format(
```

Comment:
This would maybe be a good place to use the `-C` ([docs](https://github.com/git/git/blob/5fd09df3937f54c5cfda4f1087f5d99433cce527/Documentation/RelNotes/1.8.5.txt#L115-L116)) flag rather than having to `cd` around